prow/gerrit/adapter/adapter.go

```diff
@@ -253,10 +253,13 @@ func (c *Controller) processChange(latest client.LastSyncState, changeChan <-cha
 		now := time.Now()
 
 		result := client.ResultSuccess
-		if err := c.triggerJobs(log, instance, change); err != nil {
-			result = client.ResultError
-			log.WithError(err).Info("Failed to trigger jobs based on change")
+		if !c.shouldSkipProcessingChange(instance, change, latest) {
+			if err := c.triggerJobs(log, instance, change); err != nil {
+				result = client.ResultError
+				log.WithError(err).Info("Failed to trigger jobs based on change")
+			}
 		}
+		log.Info("No relevant update in this change for Gerrit adapter, its processing is skipped.")
```

Comment:
Sure, will move this outside. The `log` on line 246 should already contain the change, revision info, etc.

Reply:
> But also, I'm a bit confused why we are logging this unconditionally (please fix?)

Ah, didn't see that (thanks GitHub for auto-hiding that...). Please do remove "Gerrit adapter" from the message.

Pasting my last question because you didn't respond: But also, I'm a bit confused why we are logging this unconditionally (please fix?)

Reply:
Would the log automatically have those info since we configure it in the log entry above?

Reply:
Moved the logging statement to else branch.